src/main.ts

```diff
@@ -10,7 +10,7 @@ const cpExec = util.promisify(require('child_process').exec);
 import { createScriptFile, TEMP_DIRECTORY, NullOutstreamStringWritable, deleteFile, getCurrentTime, checkIfEnvironmentVariableIsOmitted } from './utils';
 
 const START_SCRIPT_EXECUTION_MARKER: string = "Starting script execution via docker image mcr.microsoft.com/azure-cli:";
-const AZ_CLI_VERSION_DEFAULT_VALUE = 'agentazcliversion';
+const AZ_CLI_VERSION_DEFAULT_VALUE = 'agentazcliversion'

```

Comment:
Why is the semicolon removed?

Reply:
Since this line is not related this pr, I don't want to include this line in comparison to lead confusion.